### PR TITLE
#2865: Moves 'Toggle Camera Light' to AI Commands.

### LINKED ISSUE #2865: The AI has a "commands" tab seperate from its "AI commands" tab that has only one thing on it.
AI has a command tab that has only "turn AI lights on/off" on it, just move it to fucking AI commands tab

### PR DESCRIPTION
**Pull request by ghost**
Fixes #2865 

**Changelog:**
- tweak: "Moved 'Toggle Camera Light' to AI Commands."

### CODE REVIEW

**Comment by Spacedong** [comment]
noice

**Comment by SplergeBucket** [comment]
ok

**Comment by ghost** [comment]
Uh I guess okay +1?

**Comment by Pyko1** [comment]
Code looks fine and this is a bug fix really.

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -747,6 +747,9 @@ var/list/ai_list = list()
 	apc.malfvacate()
 
 /mob/living/silicon/ai/proc/toggle_camera_light()
```
Comment (TDcoolguy300) [inline code/modules/mob/living/silicon/ai/ai.dm:749]: Here's an issue, your pushing a proc into the AI commands tab, which is not a good. Make a verb to call the proc.

**Comment by SplergeBucket** [comment]
needs fix, do fix ty

**Comment by doobiedoo23** [comment]
+1/3

**Comment by Nebyab** [comment]
k

**Comment by Spacedong** [comment]
gonna try and put up a fix for this

**Pull request closed without merging**